rhombus/private/port.rkt

```diff
@@ -21,10 +22,22 @@
    Output
    [current_input current-input-port]
    [current_output current-output-port]
-   [current_error current-error-port]))
+   [current_error current-error-port]
+   [open_output_string Port.open_output_string]
+   [get_output_string Port.get_output_string]))
 
 (define-annotation-syntax Input (identifier-annotation #'input-port? #'()))
 (define-annotation-syntax Output (identifier-annotation #'output-port? #'()))
 
 (define-static-info-syntaxes (current-input-port current-output-port current-error-port)
   (#%function-arity 3))
+
+(define/arity Port.open_output_string
+  (case-lambda
+    [() (open-output-string)]
+    [(name) (open-output-string name)]))
```

Comment:
I think this one should also use `#:inline` and `#:primitive` for the same reason.